src/syscheckd/src/db/src/db.cpp

```diff
@@ -45,6 +45,9 @@ void DB::init(const int storage,
 
     auto rsyncHandler { std::make_shared<RemoteSync>(syncThreadPool, syncQueueSize) };
 
+    dbsync_initialize(NULL);
```

Comment:
The libraries dbsync and rsync lack of logging currently, and it's out of scope of this issue to add that feature.
That's why all the required changes to move a function pointer to this section of the code weren't implemented and simply NULL was used for now.